config.yaml

```diff
@@ -66,6 +66,10 @@ options:
     description: A list of cipher suites. This is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol. By default all the available cipher suites are supported.
     type: string
     default: ""
+  ssl_principal_mapping_rules:
+    description: A list of rules for mapping from distinguished name from the client certificate to short name.
+    type: string
+    default: "DEFAULT"
```

Comment:
**todo:** As this is a fairly esoteric, maybe worth giving an example in the description. `RULE:^CN=([a-zA-Z.0-9@-]+).*$/$1/,DEFAULT` maybe?

Reply:
I agree, this probably will help.